packages/dev/test-utils/src/events.ts

```diff
@@ -121,3 +121,85 @@ export function typeText(el: HTMLElement, value: string, opts?: ITypeOpts) {
     skipClick = true;
   }
 }
+
+export function getElementCenter(element: Element) {
+  let {left, top, width, height} = element.getBoundingClientRect();
+  return {
+    x: left + width / 2,
+    y: top + height / 2
+  };
+}
+
+/**
+ * Drag an element onto a target element, by a specific delta, or both.
+ * If target element provided, will drag to center of target element.
+ * If both provided, delta will be from center of target element.
+ */
+export async function drag(element: Element, {delta, to: targetElement, steps = 2, duration = 50, type = 'mouse'}) {
```

Comment:
How do people feel about this? I think it might be useful since we're adding a bunch of DnD stuff. As to why it something like this isn't included in testing-library: https://github.com/testing-library/user-event/issues/440

Reply:
removed from this PR since I couldn't get it to work, maybe we can add back later